src/routes/Home/components/GroupsInfo.js

```diff
@@ -2,31 +2,116 @@ import React from "react";
 import { Header, Card, Feed } from 'semantic-ui-react';
 import { Link } from 'react-router';
 import _ from 'lodash';
+import moment from 'moment';
+
+const actionFormatting = {
+  CREATED: "Ticket created",
+  PAGE_SENT: "Page sent",
+  ACKNOWLEDGED: 'Ticket aknowledged',
+  REJECTED: 'Ticket rejected',
+  CLOSED: 'Ticket closed'
+};
 
 class GroupsInfo extends React.Component {
 
+  constructor() {
+    super();
+    _.bindAll(this,
```

Comment:
You don't need to bind functions you're just rendering in. It's only needed for action handlers that are run async (like onClick)